lib/index.js

```diff
@@ -46,6 +48,17 @@ exports.init = function (env_config, callback) {
         couch.start,
         installer.install,
         plugins.load,
+        function (config, callback) {
```

Comment:
this probably wants to live in its own named function like the rest in this list?